string-pokemonize.asd

```diff
@@ -1,8 +1,8 @@
-;;;; string-pokemonize.asd
+;;;; StRiNg-pOkEmOnIzE.AsD
 
-(asdf:defsystem #:string-pokemonize
-  :description "THE missing Common Lisp string function"
-  :author "Michał \"phoe\" Herda <phoe@openmailbox.org>"
-  :license "MIT license"
-  :serial t
-  :components ((:file "string-pokemonize")))
+(AsDf:dEfSyStEm #:sTrInG-PoKeMoNiZe
+  :DeScRiPtIoN "tHe mIsSiNg cOmMoN LiSp sTrInG FuNcTiOn"
+  :aUtHoR "mIcHaŁ \"PhOe\" hErDa <PhOe@oPeNmAiLbOx.oRg>"
+  :LiCeNsE "mIt lIcEnSe"
+  :sErIaL T
+  :cOmPoNeNtS ((:fIlE "sTrInG-PoKeMoNiZe")))
```

Comment:
We have no file called `sTrInG-PoKeMoNiZe.lisp` - we have case-sensitive filesystems.
Have you built the project before submitting this PR?